### PR TITLE
Remove handle_committee_info from validator gRPC API

### PR DESCRIPTION
**Pull request by lxfind**
This API is not used and we shouldn't need it.
Also added a json-RPC dedicated type for committee for the sdk to use it.

### CODE REVIEW

**Review by mwtian — APPROVED** [review]
Fix CI?

File: crates/sui-core/src/epoch/committee_store.rs [hunk 1]
```diff
@@ -77,6 +77,16 @@ impl CommitteeStore {
             .1
     }
 
+    /// Return the committee specified by `epoch`. If `epoch` is `None`, return the latest committee.
+    pub fn get_or_latest_committee(&self, epoch: Option<EpochId>) -> SuiResult<Committee> {
```
Comment (kchalkias) [inline crates/sui-core/src/epoch/committee_store.rs:81]: Iirc, in the latest explorer screens one could see the validators per epoch + who signed each checkpoint. Is this api related to that?

Comment (lxfind) [reply crates/sui-core/src/epoch/committee_store.rs:81]: Probably?

**Review by kchalkias — APPROVED** [review]
